src/Translations.js

```diff
@@ -301,6 +310,29 @@ function PreviewBase({ document, flag = undefined, muted }) {
   )
 }
 
+function useOnChildDocumentDeletedDeleteHack(onDelete) {
```

Comment:
Is het `DeletedDelete` stukje in de naam expres?

Reply:
Nee, good catch